spec/support/request_helpers.rb

```diff
@@ -135,4 +135,17 @@ def check_forwards_locale_extension
       end
     end
   end
+
+  def check_accepts_root_path
+    context "with the root path as a base_path" do
+      let(:base_path) { "/" }
+
+      it "creates the content item" do
+        put_content_item
+
+        expect(response.status).to eq(200)
+        expect(a_request(:put, %r{.*/(content|publish-intent)/$})).to have_been_made.at_least_once
```

Comment:
Draft, live, content item or publish intent.

I want to confirm the actual path requested downstream because we already encountered a situation where it was PUTing downstream to `/content//`